tools/metricsdocs/metricsdocs.go

```diff
@@ -8,18 +8,18 @@ import (
 )
```

Comment:
This should be used the same as in these other sub operators, since it's also collected to the kubevirt/monitoring repo

Reply:
Sorry, what do you mean? you mean I can't add new `\n` lines to the section titles?